webapp/pages/index.vue

```diff
@@ -130,6 +130,11 @@ export default {
       return this.$apollo.loading || (this.posts && this.posts.length > 0)
     },
   },
+  watch: {
+    '$route.query.hashtag'() {
+      this.hashtag = this.$route.query.hashtag ? this.$route.query.hashtag : null
+    },
+  },
```

Comment:
I think there is a better way to do this... we just used `watchQuery` in the other `PR` 
have a look @Mogge https://github.com/Human-Connection/Human-Connection/pull/3271/files#diff-a15746a1a8163b3facc5cf6d0458d9e5R10

Reply:
Thanks for the tip. Much better to use watchQuery